### PR TITLE
feat: added desc

### LINKED ISSUE #72: add reference to "Apache APISIX software"
Please review and ensure that the project website meets these
requirements. http://www.apache.org/foundation/marks/pmcs

For example, I don't see a reference to "Apache APISIX
software" on either the homepage or the download page.

### PR DESCRIPTION
**Pull request by juzhiyuan**
checklist: http://www.apache.org/foundation/marks/pmcs#checklist

- [x] Desc on the homepage;
- [x] Check ASF Links at homepage's bottom.;
- [x] Check all titles are Apache APISIX;

close #72

### CODE REVIEW

**Comment by juzhiyuan** [comment]
@moonming Please help to check http://www.apache.org/foundation/marks/pmcs#metadata

File: website/siteConfig.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const users = [
 
 const siteConfig = {
   title: "Apache APISIX™",
-  tagline: "The Cloud-Native API Gateway",
+  tagline: "A dynamic, real-time, high-performance API gateway, based on the Nginx library and etcd.",
```
Comment (moonming) [inline website/siteConfig.js]:
```
Apache APISIX is a dynamic, real-time, high-performance Cloud-Native API gateway, based on the Nginx library and etcd.

Apache APISIX software provides rich traffic management features such as load balancing, dynamic upstream, canary release, circuit breaking, authentication, observability, and more.
```

Comment (moonming) [reply website/siteConfig.js]: And we need add this desc in download page too.

**Review by moonming — CHANGES_REQUESTED** [review]

File: website/siteConfig.js (hunk 1 above)
Comment (juzhiyuan) [reply website/siteConfig.js]: image

Too long to make it look better, but works.

Comment (juzhiyuan) [reply website/siteConfig.js]: image

**Review by moonming — APPROVED** [review]

**Comment by juzhiyuan** [comment]
ping @LiteSun

**Review by LiteSun — APPROVED** [review]

**Pull request merged**